config/gitleaks.toml

```
@@ -2137,6 +2137,13 @@ regex = '''glpat-[\w-]{20}'''
 entropy = 3
 keywords = ["glpat-"]
 
+[[rules]]
+id = "gitlab-pat-routable"
+description = "Identified a GitLab Personal Access Token (routable), risking unauthorized access to GitLab repositories and codebase exposure."
+regex = '''glpat-[0-9a-zA-Z_-]{27,300}\.[0-9a-z]{2}[0-9a-z]{7}'''
```

> @nwittstruck can you regenerate the config? there is a mismatch between the defined regex in the rule struct and the generated one
> 
> ```
> go generate ./...
> ```

My bad, accepting a github suggestion does not re-generate gitleaks.toml

after re-generating, your gitleaks.toml should look like this:

```suggestion
regex = '''\bglpat-[0-9a-zA-Z_-]{27,300}\.[0-9a-z]{2}[0-9a-z]{7}\b'''
```

---

Yes, it does! Thank you for providing a change suggestion!